webxrlayers-1.bs

```diff
@@ -815,7 +824,7 @@ When this method is invoked, the user agent MUST run the following steps:
 
 ISSUE: Define how cubemap sizes are determined.
 
-ISSUE: Determine if depth textures are needed for cubemap layers.
+ISSUE: How should space be handled. Can you walk to the edge of a cubemap?
```

Comment:
I agree. What type of space would this be? It's not "viewer" or "local"
I will open another issue.

Reply:
Answering my own question here:

For XrCompositionLayerCubeKHR [OpenXR's spec states:](https://www.khronos.org/registry/OpenXR/specs/1.0/html/xrspec.html#XrCompositionLayerCubeKHR):

> space is the XrSpace in which the orientation of the cube layer is evaluated over time.

Which pretty clearly indicates that only the orientation of the space is considered, and the cube will always be rendered with the view point at the center.

On the Oculus side, [the PC SDK describes an ovrLayerCube as:](https://developer.oculus.com/reference/libovr/1.43/structovr_layer_cube):

> Describes a layer of type ovrLayerType_Cube which is a single timewarped cubemap at infinity.

Which, again, is pretty explicit about the idea that position won't be considered when rendering.

Reply:
> What type of space would this be? It's not "viewer" or "local"

I believe that the intent is that you could provide any space here and the layer would only consider the orientation component of the spaces transform.

Reply:
> I believe that the intent is that you could provide any space here and the layer would only consider the orientation component of the spaces transform.

Yes. That seems to be the case. This means I should remove the checks for "viewer" from the spec.
I will talk to @Artyom17 about this since he mentioned that "viewer" space doesn't work for these layers.

Reply:
Equirect works a bit differently it seems. [OpenXR's spec](https://www.khronos.org/registry/OpenXR/specs/1.0/html/xrspec.html#XrCompositionLayerEquirectKHR) makes it clear that the equirect is rendered as an in-world object and explicitly states that the back faces aren't rendered, implying you can actually move outside it. However:

> radius is the non-negative radius of the sphere onto which the equirect image data is mapped. Values of zero or floating point positive infinity are treated as an infinite sphere.'

So you CAN create a equirect layer that effectively negates the position and only considers the space's orientation, it's just not the only way to use the layer (unlike the cube map).

The Oculus PC API does not appear to have an equivalent layer type. The [mobile SDK does](https://developer.oculus.com/reference/mobile/v15/structovr_layer_equirect2), but I can't tell from the documentation how this case is handled.